frontend/storybook/src/components/entry/popups/studentcode-paragraph.ts

```diff
@@ -0,0 +1,90 @@
+import "@elements/entry/home/TOSORT/column-details";
```

Comment:
entry/popups isn't the right location - should either be a sub-component of some other entry (like home), a common widget, or in core